prom/poller_test.go

```diff
@@ -41,13 +41,15 @@ func setup() (*url.URL, map[string]float64, func()) {
 	)
 	gaugeVec.WithLabelValues("office").Add(75)
 	gaugeVec.WithLabelValues("kitchen").Add(76)
+	gaugeVec.WithLabelValues("pantry #1").Add(71)
```

Comment:
Not in prometheus label values! That's why we were seeing "PS Scavenge" and things.